spacelift/data_policies.go

```diff
@@ -0,0 +1,74 @@
+package spacelift
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal"
+	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal/structs"
+)
+
+func dataPolicies() *schema.Resource {
+	return &schema.Resource{
+		Description: "" +
+			"`spacelift_policies` can find all policies that have certain labels," +
+			"and return their IDs. ",
+
+		ReadContext: dataPoliciesRead,
+
+		Schema: map[string]*schema.Schema{
+			"type": {
+				Type:        schema.TypeString,
+				Description: "type of the policy",
+				Required:    true,
```

Comment:
I would not mark it as required. Someone may want to match using labels instead.

Reply:
done, type is optional